core/src/main/java/nl/nn/adapterframework/pipes/FixedResultPipe.java

```diff
@@ -283,16 +251,10 @@ public void setStyleSheetName (String styleSheetName){
 	public void setLookupAtRuntime(boolean b){
 		lookupAtRuntime=b;
 	}
```

Comment:
I don't think this does anything? Perhaps the setter can be deprecated and the field can be removed.

Reply:
Used in configure to verify the existence of the file in configure time or in runtime.

Reply:
Correct me if I'm wrong but it's a classpath file. It's not like people can write a file somewhere and for this pipe to use it.. The file has to be in the jar/war.

I would expect the resource retrieval in configure (and save the URL), and when using the filenameSessionkey it would do it runtime.

Reply:
I think it is meant as a development-time option